src/main/resources/data/minecraft/tags/blocks/leaves.json

```diff
@@ -1,11 +1,6 @@
 {
+  "replace": false,
   "values": [
-    "minecraft:jungle_leaves",
-    "minecraft:oak_leaves",
-    "minecraft:spruce_leaves",
```

Comment:
Does having these in here have any negative side effects? They're in here as a workaround to a Fabric API issue (FabricMC/Fabric#66) that only occurs in dev environments.